website/content/api-docs/system/replication/replication-dr.mdx

```diff
@@ -438,13 +438,28 @@ $ curl \
 
 ## Update DR secondary's primary
 
-This endpoint changes a DR secondary cluster's assigned primary cluster using a
-secondary activation token. This does not wipe all data in the cluster.
+This endpoint changes a DR secondary cluster's assigned primary cluster. This does not
+wipe all data in the cluster.
 
 This endpoint requires a DR Operation Token to be provided as means of
 authorization. See the [DR Operation Token API
 docs](#generate-disaster-recovery-operation-token) for more information.
 
+There are two different ways to use this API call, each with its own parameter. The first way
+is using a secondary activation token (`token` in the parameter list below). This is the original
+way to call `update-primary`. This method is required for newly demoted DR secondary clusters that,
+after a DR failover, need to be pointed to their newly promoted DR primary cluster.
+
+The other way is with the `update_primary_addrs` parameter, which was added in Vault 1.13. This is an array
+of Vault cluster addresses that represent the cluster listeners of the primary cluster. These addresses will
+be pinged one at a time via gRPC and the ones that respond successfully will be used as the secondary's new
+primary addresses. This is a quicker process vs using a secondary activation token. Vault does
+less work behind the scenes, which means potentially less disruption for replication. This method
+will not work on a DR secondary cluster that is currently demoted as part of a DR failover.
+
+It's important to note that `update_primary_addrs` and `token` are mutually exclusive. You may use
+one or the other, but not both.
```

Comment:
```suggestion
1. **Use primary cluster addresses with the `update_primary_addrs` parameter**. 
    During the update, cluster addresses are pinged one at a time via gRPC.
    The first cluster to respond successfully is assigned as the new primary
    address. Updating with cluster addresses is not allowed after a DR
    failover when updating a newly demoted DR secondary cluster.

The two update methods are mutually exclusive. You may use one or the
other, but not both. We recommend updating primary cluster
assignments with cluster addresses whenever possible. Address updates
are much faster than activation token updates because Vault does less
work behind the scenes, which means potentially less disruption in
replication.
```

Style correction: write in active voice, avoid "this" as a pronoun, make key information easy to scan, focus on documenting current behavior